api/specs/webserver/openapi-projects-metadata.yaml

```diff
@@ -49,45 +49,45 @@ paths:
 components:
   schemas:
     Envelope_ProjectMetadataGet_:
+      title: Envelope[ProjectMetadataGet]
+      type: object
```

Comment:
The same happened to me with the project comments, I dont know why the ordering changed